recipes/r-shogun/meta.yaml

```diff
@@ -0,0 +1,149 @@
+{% set gpl_sha256 = "ff8a46b7775e784f07e9ffb02e7cd9b101f508c1c7b7d8cdbf1024a2efacdb22" %}
+{% set gpl_url = "http://github.com/shogun-toolbox/shogun-gpl/archive/" + gpl_commit + ".tar.gz" %}
+
+{% set blas_variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: shogun_{{ version }}.tar.gz
+  url: https://github.com/shogun-toolbox/shogun/archive/{{ commit }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  features:
+    - blas_{{ blas_variant }}  # [not win]
+  skip:
+    - true  # [win32]         # shogun doesn't build 32-bit
+    - true  # [win and py2k]  # shogun doesn't build on VC9
+    - true  # [py!=36]        # python just build dep, don't need matrix
+  script: |
+    set -ex  # not automatic in conda-build 2
+
+    # HACK: download the gpl tarball
+    # When we migrate to conda-build 3, can have multiple sources:
+    #  https://github.com/conda/conda-build/pull/1929
+    python $RECIPE_DIR/get_tarball.py  -c {{ gpl_sha256 }} -p src/gpl --strip-components 1 {{ gpl_url }}  # [unix]
+    python %RECIPE_DIR%/get_tarball.py -c {{ gpl_sha256 }} -p src/gpl --strip-components 1 {{ gpl_url }}  # [win]
+
+    mkdir build
+    cd build
+
+    cmake .. \                                # [unix]
+        -DCMAKE_PREFIX_PATH="$PREFIX" \       # [unix]
+        -DCMAKE_INSTALL_PREFIX="$PREFIX" \    # [unix]
+        -DCMAKE_BUILD_TYPE=Release \          # [unix]
+        -DBUILD_EXAMPLES=OFF \                # [unix]
+        -DBUILD_META_EXAMPLES=OFF \           # [unix]
+        -DENABLE_TESTING=OFF \                # [unix]
+        -DENABLE_COVERAGE=OFF \               # [unix]
+        -DUSE_SVMLIGHT=OFF \                  # [unix]
+        -DSWIG_EXECUTABLE=$PREFIX/bin/swig \  # [unix]
+        -DLIBSHOGUN=OFF \                     # [unix]
+        -DINTERFACE_R=ON                      # [unix]
+    make -j $CPU_COUNT                        # [unix]
+    make install -j $CPU_COUNT                # [unix]
+
+    cmake .. ^                                       # [win]
+        -G"%CMAKE_GENERATOR%" ^                      # [win]
+        -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^     # [win]
+        -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^  # [win]
+        -DCMAKE_BUILD_TYPE=Release ^                 # [win]
+        -DBUILD_EXAMPLES=OFF ^                       # [win]
+        -DBUILD_META_EXAMPLES=OFF ^                  # [win]
+        -DENABLE_TESTING=OFF ^                       # [win]
+        -DENABLE_COVERAGE=OFF ^                      # [win]
+        -DUSE_SVMLIGHT=OFF ^                         # [win]
+        -DSWIG_EXECUTABLE=%LIBRARY_BIN%\\swig.exe ^  # [win]
+        -DLIBSHOGUN=OFF ^                            # [win]
+        -DINTERFACE_R=ON                             # [win]
+    if errorlevel 1 exit 1                           # [win]
+
+    cmake --build . --target _interface_r --config Release  # [win]
```

Comment:
i think this should be `interface_r`, its only in case of python you have the `_` prepended, see:
https://github.com/shogun-toolbox/shogun/blob/develop/cmake/ShogunInterfaces.cmake#L14-L16